src/main/webapp/app/exercise/exercise-headers/exercise-header-actions/exercise-header-actions.component.html

```
@@ -0,0 +1,273 @@
+                <button
+                    id="submit-exercise"
+                    jhi-exercise-action-button
+                    [buttonIcon]="faPaperPlane"
+                    [buttonLabel]="'entity.action.submit' | artemisTranslate"
+                    [smallButton]="smallButtons()"
+                    (click)="submitAndShowPopover()"
+                ></button>
+            </span>
+        } @else {
+            <button
+                id="submit-exercise"
```

HTMLHint (1.9.2)

[error] The id value [ submit-exercise ] must be unique.

(id-unique)